packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -850,17 +850,34 @@ export function createLodestarMetrics(
     },
 
     opPool: {
-      // Note: Current opPool metrics only track current size.
-      //       I don't believe tracking total add() count is relevant since that can be seen with gossip ACCEPTs
-      aggregatedAttestationPoolSize: register.gauge({
-        name: "lodestar_oppool_aggregated_attestation_pool_size",
-        help: "Current size of the AggregatedAttestationPool = total attestations",
-      }),
-      /** This metric helps view how many overlapping attestations we keep per data on average */
-      aggregatedAttestationPoolUniqueData: register.gauge({
-        name: "lodestar_oppool_aggregated_attestation_pool_unique_data_count",
-        help: "Current size of the AggregatedAttestationPool = total attestations unique by data",
-      }),
+      aggregatedAttestationPool: {
+        aggregatedAttestationPoolSize: register.gauge({
```

Comment:
since those metrics are now part of `aggregatedAttestationPool` namespace I would suggest more brief names

```suggestion
        size: register.gauge({
```
(same for the other one below)